src/App.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './App.css';
+
+import resumeJSON from "./data/resume"
+import { generateResume } from "./schemaModels/process"
+
+import Introduction from "./components/Introduction"
+import Experience from "./components/Experience"
+import Education from "./components/Education"
+import Volunteer from "./components/Volunteer"
+
+import Divider from "./components/shared/Divider"
+
+function App() {
+  const resume = generateResume(resumeJSON)
+
+  return (
+    <div className="App">
+      <div className="resumeContainer">
```

Comment:
Hi, I believe that naming convention best practice for classes in React is to use kebab case, i.e. `className="resume-container"`